lib/bike_club.rb

```diff
@@ -1,3 +1,29 @@
+
+
 class BikeClub
+    attr_accessor :bikers
+    attr_reader :name, :bikers
+
+    def initialize(name)
+        @name = name
+        @bikers = []
+    end
+
+    def add_biker(biker)
+        @bikers << biker
+    end
+    
+    #checks bikers rides by accessing @bikers, and returns the biker with the most rides. max_by is a method that returns the element that has the maximum value of the block.
+    def most_rides
+        @bikers.max_by do |biker|
```

Comment:
So close! This will only count the number of key-value pairs in the `rides` hash, but we want the number of time elements in the value arrays. So we want to do `biker.rides.values.flatten.length` to get those (like we did in the walkthrough today!)